sagemaker_edge_manager/sagemaker_edge_example/sagemaker_edge_example.ipynb

```diff
@@ -38,22 +38,22 @@
     "\n",
```

Comment:
<p>I see it's explained above in the text, but add a comment right in the cell instructing the customer to fill in the version as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='90'/>

Reply:
Sure